Sources/Testing/Testing.docc/Traits/Trait.md

```diff
@@ -39,8 +39,15 @@ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 - ``Trait/bug(_:id:_:)-3vtpl``
 
 ### Adding information to tests
+
 - ``Trait/comments``
 
 ### Preparing internal state
 
 - ``Trait/prepare(for:)-3s3zo``
+
+### Customizing the execution of tests
+
+- ``CustomTestExecuting``
+- ``Trait/customTestExecutor-1dwpt``
```

Comment:
Is this showing up as a separate symbol in DocC? Is the DocC bundle building cleanly with no warnings here?

Reply:
It's because it has multiple overloads in extensions

Reply:
Yes, there are no (new) DocC diagnostics

Reply:
Ah, okay.